src/engraving/libmscore/tuplet.cpp

```diff
@@ -485,6 +485,24 @@ void Tuplet::setTrack(track_idx_t val)
     EngravingItem::setTrack(val);
 }
 
+//---------------------------------------------------------
+//   nextElement
+//---------------------------------------------------------
+
+EngravingItem* Tuplet::nextElement()
+{
+    return elements().front();
+}
+
+//---------------------------------------------------------
+//   prevElement
+//---------------------------------------------------------
+
+EngravingItem* Tuplet::prevElement()
+{
+    return elements().back();
```

Comment:
The previous element will be something before the tuplet, not something inside the tuplet. Exceptionally, it might be an element in a different voice that occurs at the same time as the tuplet.

You probably need to call `Segment::previousElement()` or something like that.